src/components/Sum.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Sum(props) {
```

Comment:
Делай деструктуризацию props
```suggestion
function Sum({ a, b }) {
```